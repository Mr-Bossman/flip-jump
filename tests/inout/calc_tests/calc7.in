23-100
q
