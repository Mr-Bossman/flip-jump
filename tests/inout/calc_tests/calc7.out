>  23-100
-77
>  q
